test/release/examples/benchmarks/shootout/chameneosredux-fast.chpl

```diff
@@ -35,9 +35,6 @@ proc main() {
 
   print(group1);
   print(group2);
-
-  for c in group1 do delete c;
-  for c in group2 do delete c;
```

Comment:
Should this be removed? Looks like they're still unmanaged above. Oh, maybe missed in the reversion in 3d822dd55ddb389d303719813fabd92262bba6e0?

Reply:
I tried this, but got a deadlock which was surprising to me.  I just sent you and @mppf mail about it in hopes that he/you might have insights.

Reply:
Sorry --I meant that it's still using unmanaged, but you removed the deletes (i.e. I think we're leaking now.)

Reply:
Oh, I see.  Huh, could've sworn that I added those back in.  Maybe I forgot to git add...  Thanks!  (my memleaks run hasn't finished yet, but will presumably show this too).